src/compiler.js

```diff
@@ -3,6 +3,32 @@
 var BuildMail = require('buildmail');
 var libmime = require('libmime');
 
+// HACK(rtasson): the following couple of lines are hacks to permit us to
+// modify the MIME part boundary prefix without upgrading our nodemailer
+// version to one that supports this natively.
+// The following baseBoundaryPrefix is what nodemailer currently uses:
+// https://github.com/nodemailer/nodemailer/blob/9b5fb94767c6d9ba0851dc487b5b4a2842cdae75/lib/mime-node/index.js#L44
+let baseBoundaryPrefix = '--_NmP';
```

Comment:
For more backward compatibility assurances, you could leave this as '----sinikael-?=_' by default and let clients set it as needed.